src/containers/Experiment/SamplesTable.js

```diff
@@ -277,3 +282,80 @@ function ThComponent({ toggleSort, className, children, ...rest }) {
     </div>
   );
 }
+
+function ProcessingInformationCell({ original: sample, ...props }) {
+  let { annotations } = sample;
+
+  // If the sample is marked with `is_ccdl = false` then this sample is "Submitter Processed"
+  if (annotations.length > 0 && !annotations[0].is_ccdl) {
```

Comment:
it's always the first annotation on a sample?

Reply:
This is not yet clear to me... I added this condition for now, the logic to decide which dialog to display will be added with https://github.com/AlexsLemonade/refinebio-frontend/pull/50

 We have to discuss what happens when samples have more than one annotation.